src/xradio/image/_util/_casacore/xds_to_casacore.py

```diff
@@ -201,7 +201,7 @@ def _history_from_xds(xds: xr.Dataset, image: str) -> None:
 def _imageinfo_dict_from_xds(xds: xr.Dataset) -> dict:
     ii = {}
     ap_sky = _aperture_or_sky(xds)
-    ii["image_type"] = (
+    ii["imagetype"] = (
```

Comment:
Although "image_type" is written into the "imageinfo" key, `image::imageinfo()` can not retrieve it. The issue was probably not exposed in the past tests using `python-casacore` as `image::imageinfo()` would fallback and still return `{'imagetype': 'Intensity', 'objectname': ''}`, as shown here:


```python
In [1]: from casacore import tables

In [2]: tb=tables.table('from_xds_no_sky.im')
Successful readonly open of default-locked table from_xds_no_sky.im: 1 columns, 1 rows

In [3]: tb.getkeyword('imageinfo')
Out[3]: {'image_type': 'Intensity', 'objectname': ''}

In [5]: from casacore import images

In [7]: im=images.image('from_xds_no_sky.im')

In [8]: im.imageinfo()
Out[8]: {'imagetype': 'Intensity', 'objectname': ''}
```